internal/cephfs/clone.go

```diff
@@ -43,6 +43,28 @@ const (
 	snapshotIsProtected = "yes"
 )
 
+// toError checks the state of the clone if it's not cephFSCloneComplete.
+func (cs cephFSCloneState) toError(ctx context.Context, cloneID volumeID) error {
```

Comment:
btw, this function is ideal to be verified by a unit test. Please consider adding one.

Reply:
I suggest that we can open another issue for it and it can be done there.

Reply:
we can, but for most cases where we add new helper functions, we also add unit tests. It is just that for many existing functions no tests were written :disappointed:

Reply:
No worries, I will add the unit test too. I have created an issue for that.

Reply:
@yati1998 as Niels mentioned the process we follow is to add unit testing in the same commit which introduced the new function (this will help us to verify that we getting expected output from the function or not). Please do consider this for next time.

Reply:
okay. Sure @Madhu-1 .